### PR TITLE
[MM-43638] Add placeholder to channel members search

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
This PR adds a placeholder to channel members search

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-43638

#### Related Pull Requests
N/A

#### Screenshots
image

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/widgets/inputs/input/input.tsx [hunk 1]
```diff
@@ -98,9 +100,11 @@ function Input({
                     Input_fieldset___legend: showLegend,
                 })}
             >
-                <legend className={classNames('Input_legend', {Input_legend___focus: showLegend})}>
-                    {showLegend ? label || placeholder : null}
-                </legend>
+                {!noLegend && (
```
Comment (pvev) [inline components/widgets/inputs/input/input.tsx:103]: nit: better call it addLegend and prevent the negation condition

**Review by pvev — APPROVED** [review]
LGTM! nice work @JulienTant  :shipit:

File: components/widgets/inputs/input/input.tsx (hunk 1 above)
Comment (julmondragon) [reply components/widgets/inputs/input/input.tsx:103]: Perhaps `useLegend`.

**Review by matthewbirtch — APPROVED** [review]

**Review by stevemudie — APPROVED** [review]
Looks good and works great @JulienTant!

**Pull request merged**